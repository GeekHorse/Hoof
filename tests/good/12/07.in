new right one done
new in subone done
new in subsubonea done
out
out
most in
new down subsuboneb done
save
