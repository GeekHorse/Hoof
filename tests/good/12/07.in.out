new right one done
new in subone done
new in subsubonea done
new down subsuboneb done
out
out
